ui/decredmaterial/card.go

```diff
@@ -11,32 +11,49 @@ import (
 
 type Card struct {
 	layout.Inset
-	Color   color.RGBA
-	Rounded bool
+	Color  color.RGBA
+	Radius CornerRadius
+}
+
+type CornerRadius struct {
+	NE int
+	NW int
+	SE int
+	SW int
```

Comment:
this can be a float32 type. Removes the need to type it to float32 when specifying the dimensions for the rectangle.